src/components/Composer.vue

```diff
@@ -278,6 +292,7 @@ const STATES = Object.seal({
 	SENDING: 2,
 	ERROR: 3,
 	FINISHED: 4,
+	WARNING: 5,
```

Comment:
I think that should be moved up and FINISHED should stay last

Reply:
So `WARNING: 4` and `FINISHED: 5`?

Reply:
yep :)